pennylane/measurements.py

```diff
@@ -846,6 +862,42 @@ def circuit(x):
     )
 
 
+def mutual_info(wires0, wires1, log_base=None):
+    r"""Mutual information between the subsystems prior to measurement.
```

Comment:
Maybe you can add: It is the measure of correlation between the subsystems.

Reply:
Added the extra description.